tests/integration/cgroups.bats

```diff
@@ -71,6 +72,24 @@ EOF
     fi
 }
 
+@test "runc update --memory (uninitialized)" {
+    [[ "$ROOTLESS" -ne 0 ]] && requires rootless_cgroup
+    requires cgroups_v2
+
+    set_cgroups_path "$BUSYBOX_BUNDLE"
+
+    # run a detached busybox to work with
+    runc run -d --console-socket $CONSOLE_SOCKET test_cgroups_mem
+    [ "$status" -eq 0 ]
+
+    # update memory limit
+    runc update test_cgroups_mem --memory 50331648
+
+    # check value of memory.max
+    [ "$status" -eq 0 ]
+    check_cgroup_value "memory.max" 50331648
```

Comment:
Just compare `/sys/fs/cgroup/cgroup.controllers`.
On rootless systemd, the content should correspond to `/sys/fs/cgroup/user.slice/user-${uid}.slice/cgroup.controllers`